terraform/modules/upload-service/variables.tf

```diff
@@ -25,6 +25,11 @@ variable "ingest_api_key" {
   type = "string"
 }
 
+// Checksum Lambda
```

Comment:
Please set a default here since its the same for all environments currently. If it does change in the future, this still gives us the option to set the appropriate image value in the respective environment tfvars file

Reply:
👍